pages/platform/public-cloud/using_openstack_tokens_for_multiple_calls/guide.en-gb.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Using Openstack Tokens"
+slug: using-openstack-tokens
+excerpt: "Find out how to create and use Openstack tokens for your actions"
+section: "Openstack"
+order: 01
+updated: 2023-05-02
+---
+
+**Last updated 2nd May 2023**
+
+## Objective
+
+**This guide provide good practices to perform a lot of Openstack actions in a short time**
+
+> [!primary]
+>
+Informations in this guide applies to version 3.0 of the Keystone API
+>
+
+## Instructions
+
+### Definitions
+
+- Endpoint: HTTP address pointing directly to a service's API. For example [https://auth.cloud.ovh.net/v3/](https://auth.cloud.ovh.net/v3/) for the authentication endpoint or [https://image.compute.gra11.cloud.ovh.net/](https://image.compute.gra11.cloud.ovh.net/) for the GRA11 zone image management endpoint. 
+
+
+- Token: A unique string of characters used to authenticate and access resources. The user requests it by entering their credentials (login details) to the authentication API. The token is generated and it is valid for 24 hours.
+
+- OpenRC : To increase efficiency of interactions with the Identity service via the openstack client, OpenStack supports simple client environment scripts also known as OpenRC files.
+This is a file containing common options for all clients, but also support unique options.
+
+### Outline of a request
+Most requests sent to the OpenStack API must follow an authorization procedure, which involves generating a token and validating it.
+
+However, if you perform too much actions in a short time, some Openstack actions will fall in error due to too much API Calls
```

Comment:
Maybe we can give more information here, like (to be better rephrase) :
The interest reuse a token avoid the cli create one at each command run, make it quicker, and avoid HTTP 429  error because token generation is rate limited to 60 per minutes, and other API call does not have such limit.